src/fsharp/vs/IncrementalBuild.fsi

```diff
@@ -279,3 +279,6 @@ module internal IncrementalBuild =
         /// Set the conrete inputs for this build. 
         member GetInitialPartialBuild : vectorinputs: BuildInput list -> PartialBuild
 
+type internal CompilationGlobalsScope =
```

Comment:
Please add a `///` comment (or copy the one from the implementation file if present)